From 917b7fb08a18d421df4e627fe33f6335989384fd Mon Sep 17 00:00:00 2001
From: Vitaly Margolin <vmargolin@habana.ai>
Date: Tue, 06 May 2025 15:30:29 +0300
Subject: [PATCH] [SW-228057] lfs: add serdes lfs part. csum to fw info

Add Serdes LFS partition sha384 checksum to fw_info,
so that arcmgmt will be able to check it's validity

Change-Id: Iaefcf6c4f8f22f9e5b3331a8b6890a4ef8e5fe5b
Signed-off-by: Vitaly Margolin <vmargolin@habana.ai>
---

diff --git a/hl_fw_if.h b/hl_fw_if.h
index c7a5a9a..83ebab3 100644
--- a/hl_fw_if.h
+++ b/hl_fw_if.h
@@ -135,6 +135,7 @@
 	uint16_t reserved2;
 	uint32_t serdes_lfs_partition_offset;	/*The offset in flash of serdes data partition */
 	uint32_t serdes_lfs_partition_size;	/*The size of serdes data partition */
+	uint8_t serdes_lfs_partition_checksum[SHA384_SUM_LEN];
 } __packed;
 
 
